### PR TITLE
new quotes

### PR DESCRIPTION
**Pull request by Pato-desu**

### CODE REVIEW

**Comment by Pato-desu** [comment]
whaaat the places of the images change depending on the text? then I don't have idea how to fix some of them. @joepio fix them when you can

**Comment by Pato-desu** [comment]
Btw:
- the images ARE NOT checked for copyright. Is that an important thing? I dont know how to check it
- some images and text I didnt take too much time to select them and edit them, so i wouldnt be surprised if there are better
-  this has the same conflict than the other PR, i just noticed
- trump being at the top it is in purpose for bipartisan support, but idk, im open to hear other opinions

**Comment by joepio** [comment]
The images do need to be CC0 or checked before I can accept them I'm afraid

**Comment by Pato-desu** [comment]
Okay, Ill see how can I do that

**Comment by Pato-desu** [comment]
Is that the only type? you put a note in the Turing quote for other type of licenses, right?

**Comment by Pato-desu** [comment]
@joepio let's say I don't find a CC picture of Paul Christiano, should I take the quote off the page?

**Comment by mrbreastly** [comment]
Also, check preview here...  https://deploy-preview-125--pauseai.netlify.app/quotes
- Lots of images are way off center... (use `"padding": "#rem"` to align the text and images... e.g.     "padding": "3.6rem",     "padding": "1rem", etc)
- Text color is often unreadable (e.g. use    `"color": "black"` or `"color": "white"` depending on BG color.. etc

image
image

**Comment by mrbreastly** [comment]
> * trump being at the top it is in purpose for bipartisan support, but idk, im open to hear other opinions

Yeah, might be worth including "Donald Trump, 45th President of the United States", "Joe Biden, 46th President of the United States"... so they're shown to be in chronological order?

**Comment by Pato-desu** [comment]
> > whaaat the places of the images change depending on the text? then I don't have idea how to fix some of them. @joepio fix them when you can
> 
> Yeah, check preview here... https://deploy-preview-125--pauseai.netlify.app/quotes
> 
> * Lots of images are off center... (use `"padding": "#rem"` to align the text and images... e.g. try `"padding": "3.6rem"`,     `"padding": "1rem"`, etc)
> * Text color is often unreadable (e.g. use    `"color": "black"` or `"color": "white"` depending on BG color..)
> 
> image image

Yeah, but first of all I have to find pictures without copyright

File: src/routes/quotes/data.json [hunk 1]
```diff
@@ -1,25 +1,139 @@
...
+  {
+    "background": "hawking",
+    "text": "The development of full artificial intelligence could spell the end of the human race.",
+    "author": "Stephen Hawking",
+    "author_description": "Theoretical Physicist and Cosmologist",
+    "color": "white"
+  },
+  {
+    "background": "sunak",
+    "text": "Get this wrong, and AI could make it easier to build chemical or biological weapons. Terrorist groups could use AI to spread fear and destruction on an even greater scale. Criminals could exploit AI for cyber-attacks, disinformation, fraud, or even child sexual abuse. And in the most unlikely but extreme cases, there is even the risk that humanity could lose control of AI completely through the kind of AI sometimes referred to as ‘super intelligence’. Indeed, to quote the statement made earlier this year by hundreds of the world’s leading AI experts: “Mitigating the risk of extinction from AI should be a global priority alongside other societal-scale risks such as pandemics and nuclear war”.",
```
Comment (mrbreastly) [inline src/routes/quotes/data.json:19]: Awesome quote!  But, might be a little to long to fit on these small cards?  
- Maybe add a [..] to drop a little from the middle somewhere?
- Or, try to use padding to shift it around?

image

File: src/routes/quotes/data.json [hunk 2]
```diff
@@ -1,25 +1,139 @@
...
+    "background": "pope",
+    "text": "I urge the global community of nations to work together in order to adopt a binding international treaty that regulates the development and use of artificial intelligence in its many forms. [...] [AI] may pose a risk to our survival and endanger our common home",
+    "author": "Pope Francis",
+    "author_description": "Head of the Catholic Church",
+    "color": "white",
+    "padding": "0rem"
+  },
+  {
+    "background": "kamala",
+    "text": "[...] just as AI has the potential to do profound good, it also has the potential to cause profound harm. From AI-enabled cyberattacks at a scale beyond anything we have seen before to AI-formulated bio-weapons that could endanger the lives of millions, these threats are often referred to as the “existential threats of AI” because, of course, they could endanger the very existence of humanity. These threats, without question, are profound, and they demand global action.",
```
Comment (mrbreastly) [inline src/routes/quotes/data.json:79]: These two might be better with white?

image

**Review by mrbreastly — CHANGES_REQUESTED** [review]

File: src/routes/quotes/data.json [hunk 3]
```diff
@@ -1,25 +1,139 @@
...
+  {
+    "background": "leyen",
+    "text": "[We] should not underestimate the real threats coming from AI. [Fully quoted the above statement on the risk of extinction.] [...] It is moving faster than even its developers anticipated. [...] We have a narrowing window of opportunity to guide this technology responsibly.",
+    "author": "Ursula von der Leyen",
+    "author_description": "Head of the executive branch of the European Union",
+    "color": "white"
+  },
+  {
+    "background": "guterres",
+    "text": "AI poses a long-term global risk. Even its own designers have no idea where their breakthrough may lead. I urge [the UN Security Council] to approach this technology with a sense of urgency. Unforeseen consequences of some AI-enabled systems could create security risks by accident. Generative AI has enormous potential for good and evil at scale. Its creators themselves have warned that much bigger, potentially catastrophic and existential risks lie ahead. Without action to address these risks, we are derelict in our responsibilities to present and future generations.",
```
Comment (mrbreastly) [inline src/routes/quotes/data.json:63]: This one might be too long too?  Would be more readable if the text didn't overlap with the image?

image

**Review by mrbreastly — CHANGES_REQUESTED** [review]

**Comment by mrbreastly** [comment]
Hey @Pato-desu, good quotes! I made some suggestions here.  I don't have access to update this branch directly. If I get some time later I'll see if I can make a fork of your branch here and apply some fixes and a new PR, for this branch...

**Comment by Pato-desu** [comment]
Things left to do:
- [ ] Add some more quotes. Mainly from the CEOs
- [ ] Add more "placeholder" pictures like the one with the chinese flag
- [ ] Maybe add more smooth transitions between quotes and pictures

**Pull request merged**